examples/multicsc/plot_sample_evoked_response.py

```diff
@@ -264,6 +264,34 @@
 print("\rDisplayed {} atoms".format(len(plotted_atoms)).rjust(40))
 fig.tight_layout()
 
+###############################################################################
+# Here is a different visualization of the atoms correlated with the event
+# onsets. We first select two atoms based on previous figure. Each event
+# corresponds to a column. Then we pick the channel with the largest absolute
+# value in the spatial map of selected atoms. The top row displays the raw
+# timecourses on this channel, aligned on the event onsets, and ordered by
+# minimum raw value. The second row plots the average evoked timesourse,
```

Comment:
```suggestion
# minimum raw value. The second row plots the average evoked time-course,
```

Also, you seem to be aligning based on the `pred` value and not on the raw value. something like:
```
# ... The top row displays the raw
# time-courses on this channel, aligned on the event onsets, and ordered by
# the localization of the minimum value of the signal reconstructed with the
# two selected atoms. 
```